v2.1/scalar-expressions.md

```diff
@@ -701,6 +701,8 @@ specified explicitly using a type annotation. For example:
 
 {{site.data.alerts.callout_success}}To convert the results of a subquery to an array, use <a href="#conversion-of-subquery-results-to-an-array"><code>ARRAY(...)</code></a> instead.{{site.data.alerts.end}}
 
+{{site.data.alerts.callout_success}}CockroachDB also recognizes the syntax <code>ARRAY(a, b, c)</code> as an alias for <code>ARRAY[a, b, c]</code>. This is an experimental, CockroachDB-specific SQL extension and may be removed in a later version of CockroachDB.{{site.data.alerts.end}}
```

Comment:
We recently realized that, if you put the opening and closing tags on their own lines, you can use markdown formatting instead of html, for example:

```
{{site.data.alerts.callout_success}}
CockroachDB also recognizes the syntax `ARRAY(a, b, c)` as an alias for `ARRAY[a, b, c]`. This is an experimental, CockroachDB-specific SQL extension and may be removed in a later version of CockroachDB.
{{site.data.alerts.end}}
```